frontend/package.json

```diff
@@ -18,7 +18,7 @@
     "@mdi/font": "^4.9.95",
     "@sap-devx/inquirer-gui": "0.0.31",
     "@sap-devx/inquirer-gui-login-plugin": "0.0.1",
-    "@sap-devx/inquirer-gui-remote-file-browser-plugin": "0.0.1",
+    "@sap-devx/inquirer-gui-remote-file-browser-plugin": "0.0.2",
```

Comment:
Why do I have to specific declare dependency to inquirer-gui-remote-file-browser-plugin / inquirer-gui-login-plugin ?
I would expect all plugin to be available within -> "@sap-devx/inquirer-gui"

Reply:
It's a matter of definition. Currently only controls for [built-in inquirer prompt types](https://github.com/SBoudrias/Inquirer.js#prompt-types) are provided by default in inquirer-gui.  

We could change that behavior and provide custom controls (inquirer-gui plugins) that we develop together with the package, but that's a matter that product should decide.

Reply:
OK. we'll considure this in future.